src/ApplicationLogs/ApplicationLogs.csproj

```diff
@@ -5,6 +5,17 @@
     <RootNamespace>Neo.Plugins</RootNamespace>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
```

Comment:
It could be done in `Directory.Build.props`?

Reply:
> It could be done in `Directory.Build.props`?

Well, let's find out

Reply:
@shargon unfortunately, I did not figure out a way to do that.